debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/spi/Snapshotter.java

```diff
@@ -44,6 +44,16 @@ void init(PostgresConnectorConfig config, OffsetState sourceInfo,
      */
     boolean shouldStream();
 
+    /**
+     *
+     * @return true if when a connector resumes and takes a snapshot, streaming should resume from
+     * where streaming left off, or false if streaming should resume from the start of the snapshot
+     * transaction.
+     */
+    default boolean shouldStreamEventsStartingFromSnapshot() {
+        return true;
```

Comment:
I think the method JavaDoc on true/false is exactly the opposite of the implemented behavior, right?

Reply:
Oof! Yea you're right, I'll fix this.